TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -289,6 +289,17 @@ public static void OnCodeElement(StartStatement statement, CodeElementsParser.St
         }
     }
 
-    #endregion
+    class AlterStatementChecker
+    {
+        public static void OnCodeElement(AlterStatement statement, CodeElementsParser.AlterStatementContext context)
+        {
+#if EUROINFO_RULES
+            DiagnosticUtils.AddError(statement, "ALTER \"should\" not be used", context);
+#else
+            DiagnosticUtils.AddError(statement,   "ALTER \"should\" not be used", context, MessageCode.Warning);
+#endif
```

Comment:
Avoid using quotes in the messages. Use "ALTER statement is forbidden" in EI context and "ALTER statement should not be used" in general context.

Reply:
Quotes are now removed; the message is the same : code pushed